CHANGELOG.md

```diff
@@ -163,6 +163,7 @@ be used to retrieve the actual proposal `Content`. Also the `NewMsgSubmitProposa
 
 ### Features
 
+* [\#6755](https://github.com/cosmos/cosmos-sdk/pull/6755) Add custom regex validation for `Coin` denom by overwriting `CoinDenomRegex` when using `/types/coin.go`.
```

Comment:
@okwme this should be moved across the the Launchpad branch as 0.39.2 will be the first release where this new feature is introduced.